lib/rules/at-rule-descriptor-no-unknown/__tests__/index.mjs

```diff
@@ -153,5 +155,52 @@ testRule({
 				},
 			],
 		},
+		{
+			code: stripIndent`
+				@mixin --qux-baz(@contents) {
+					@supports (overflow-anchor: auto) {
+						@contents;
+					}
+				}
+
+				body {
+					@apply --qux-baz {
+						@font-face {
+						  foo: normal;
+						}
+					}
+				}
+			`,
+			message: messages.rejected('@font-face', 'foo'),
+			line: 10,
+			column: 5,
+			endLine: 10,
+			endColumn: 8,
+			description: 'without indirection',
+		},
+		{
+			code: stripIndent`
+				@mixin --qux-baz(@contents) {
+					@supports (overflow-anchor: auto) {
+						@font-face {
+							@contents;
+						}
+					}
+				}
+
+				body {
+					@apply --qux-baz {
+						foo: normal;
+					}
+				}
+			`,
+			message: messages.rejected('@font-face', 'foo'),
+			line: 11,
+			column: 3,
+			endLine: 11,
+			endColumn: 6,
+			description: 'with indirection',
+			skip: true,
```

Comment:
[suggestion] Please leave a reason for `skip: true`.

Reply:
I thought the description was enough; will do.

Reply:
```suggestion
			// TODO: unskip once we support @mixin indirection
			skip: true,
```